core/src/test/java/com/xmartlabs/base/core/observers/MaybeMainObserversWithErrorHandlingTest.java

```diff
@@ -0,0 +1,178 @@
+package com.xmartlabs.base.core.observers;
+
+import com.xmartlabs.base.core.rx.error.MaybeObserverWithErrorHandling;
+
+import org.junit.Test;
+
+import io.reactivex.Maybe;
+import io.reactivex.plugins.RxJavaPlugins;
+import timber.log.Timber;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+public class MaybeMainObserversWithErrorHandlingTest extends MainObserversWithErrorHandlingTest {
+  @Override
+  @SuppressWarnings("unchecked")
```

Comment:
This is because `new MaybeObserverWithErrorHandling<>(MaybeObserver, getErrorHandlingAction())` is shown as an unchecked type cast call to MaybeObserver...
The unchecked category allows suppression of compiler warnings generated as a result of unchecked type casts, when we know everything is fine.